### DISCUSSION TITLE
recipes/desserts/chocolate-whoopie-pies-with-vanilla-buttercream-filling/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#637] [Comments]
# recipes/desserts/chocolate-whoopie-pies-with-vanilla-buttercream-filling/

A collection of my recipes.

https://nicholaswilde.io/recipes/desserts/chocolate-whoopie-pies-with-vanilla-buttercream-filling/

**Comment by nicholaswilde** [comment]
Look at having to preheat the pans before adding the cookie mixture. https://www.youtube.com/watch?v=Zo3qlxyZabA
